test/jasmine/specs/global-object-css-spec.js

```diff
@@ -12,6 +12,24 @@ describe("Test getting JSON from <body/> CSS", function () {
         );
     });
 
+    it("should get expected strings from global element", function () {
```

Comment:
Yep, all the tests passed and I got rid of all the hacks I had in the previous implementation.